otter/integration/tests/test_convergence.py

```diff
@@ -67,6 +69,66 @@ def _check_fds(_):
             return deferLater(reactor, 0, _check_fds, None)
         return self.pool.closeCachedConnections().addBoth(_check_fds)
 
+    def test_scale_over_group_max_after_metadata_removal(self):
+        """
+        Attempt to scale over the group max, but only after metadata removal on
+        some random sampling of servers.
+        """
+
+        rcs = TestResources()
+
+        scaling_group_body = create_scaling_group_dict(
+            image_ref=image_ref, flavor_ref=flavor_ref,
+        )
+
+        self.scaling_group = ScalingGroup(
+            group_config=scaling_group_body,
+            pool=self.pool
+        )
+
+        self.scale_up_to_max = ScalingPolicy(
+            scale_by=25,
+            scaling_group=self.scaling_group
+        )
+
+        self.scale_beyond_max = ScalingPolicy(
+            scale_by=5,
+            scaling_group=self.scaling_group
+        )
+
+        return (
+            self.identity.authenticate_user(
+                rcs,
+                resources={
+                    "otter": ("autoscale", "http://localhost:9000/v1.0/{0}"),
+                    "nova": ("cloudServersOpenStack",),
```

Comment:
These values should not be hard-coded in the test. It will become difficult to run these against prod/staging environments. The resources should be loaded based on environment variables / cmd-line args when the test suite starts.